### PR TITLE
Add 'Soon' page

### PR DESCRIPTION
**Pull request by Jack-Development**
Will implement the "/soon" page
Has a countdown to the CSS Ball

### CODE REVIEW

**Comment by JTWWilson** [comment]
No counter seems to show up on the preview? 
image

File: soon.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<div class="soon-contain">
+    <h1>Soon.™</h1>
+    <h2 id="count"></h2>
+    <img src="https://raw.githubusercontent.com/CSSUoB/resources/db2bbb3b4f0b19a1099cb0fb72a2b19fdfa89999/dinosaur/tex_ball.svg" alt="TuX TeX"/>
+</div>
+
+
+<script>
+    var countDownDate = new Date("Apr 28, 2021 00:00:00").getTime();
```
Comment (AlexJBanks) [inline soon.html:16]: "2021" -> "2022"

**Review by AlexJBanks — CHANGES_REQUESTED** [review]

**Comment by AlexJBanks** [comment]
@Jack-Development 

Text needs adjusting for mobile view

File: css/soon.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.soon-contain {
+  text-align: center;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  h1 {
+    text-align: center;
+    font-size: 180px;
```
Comment (AlexJBanks) [inline css/soon.scss:13]: 180px -> 20vw

**Review by AlexJBanks — CHANGES_REQUESTED** [review]

**Review by AlexJBanks — APPROVED** [review]
Looks great to me. thanks @Jack-Development

**Pull request merged**